client/constants/messages.js

```diff
@@ -57,7 +57,7 @@ const messages = {
    * with new records, do
    * GET_EXISTING_OBJECTS -> RESOLVE_SYNC_RECORDS -> RESOLVED_SYNC_RECORDS
    */
-  FETCH_SYNC_RECORDS: _, /* @param Array.<string> categoryNames, @param {number} startAt (in seconds or milliseconds), @param {number=} maxRecords limit response to a given max number of records. set to 0 or falsey to not limit the response */
+  FETCH_SYNC_RECORDS: _, /* @param Array.<string> categoryNames, @param {number} startAt (in seconds or milliseconds), @param {number=} maxRecords limit response to a given max number of records. set to 0 or falsey to not limit the response, @param {number} previousFetchTime (in seconds or milliseconds) */
```

Comment:
not specifying whether the value is seconds or milliseconds seems like a really bad idea to me, what is the reason for this?

Reply:
@bridiver , seconds or milliseconds as a requirement of output were introduced long time ago , https://github.com/brave/sync/commit/349592a81c24c22a9c41bc9db28de1730ed6ca04#diff-e10a265c86f058932ac016e27462b9a5R54 , but we don't know what was the reason.

Reply:
That's fine for existing params, but I don't think we should continue the practice for new ones. I think it's an extremely bad idea to pass a timestamp that doesn't specify one of the other

Reply:
@bridiver addressed in https://github.com/brave/sync/pull/363/commits/c4a0e66153724a005c1d02d36d12ad308a1a50dc